src/app/client/src/app/modules/shared/components/offline-card/offline-card.component.html

```diff
@@ -67,24 +67,38 @@ <h4 class="sb-card-title">
     <sui-progress class="ui progress" [value]="data.progress" [maximum]="data.maxCount" *ngIf="data.maxCount && data.progress > 0"></sui-progress>
   </div>
   <!--/Progress Bar-->
```

Comment:
More logic in template , move it to component like **status==='OFFLINE' && data.youtubeContent === true** can be a simple method which can be used in all the places in template